pkg/api/component_report.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	triagedIncidentsTableID = "triaged_incidents"
 
-	ignoredJobsRegexp = `-okd|-recovery|aggregator-|alibaba|-disruptive|-rollback|-out-of-change|-sno-fips-recert`
+	ignoredJobsRegexp = `-okd|-recovery|aggregator-|alibaba|-disruptive|-rollback|-out-of-change|-sno-fips-recert|periodic-ci-ComplianceAsCode-|efs|interop`
```

Comment:
Actually ken that efs string looks dangerous, could that not pick up a lot of unintended jobs?

Reply:
Only one hit in sippy: pull-ci-openshift-aws-efs-csi-driver-operator-master-operator-e2e-extended
Would prefer -efs- though just to be safe, maybe same for -interop-.

Reply:
Done